Backpack-SwiftUI/AppSearchModal/Classes/BPKAppSearchModalResults.swift

```diff
@@ -112,6 +112,18 @@ public struct BPKAppSearchModalAction {
     }
 }
 
+extension BPKAppSearchModalContent.Shortcut: Equatable {
```

Comment:
Could you add conformance to Equatable in the struct definition instead? so you dont have to implement this manually (which we can forget to update if we add a new property! )